Automattic/jetpack
# Blogroll: Update styling

## Proposed changes:



Changes related to these issues:

https://github.com/Automattic/jetpack/issues/37927 - Removes left/right padding on blogroll-item
https://github.com/Automattic/jetpack/issues/37929 - Update title to match paragraph font-size and add bold styling
https://github.com/Automattic/jetpack/issues/37928 - Aligns avatar to the top of blogroll-item
https://github.com/Automattic/jetpack/issues/37930 - Match blogroll-item divider with theme preset color and set opacity to 20%

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Sync with simple and atomic site 
* Verify changes with the issues listed above
